common/envVars.go

```diff
@@ -43,4 +43,7 @@ const (
 	// ArgoCDClusterConfigNamespacesEnvVar is the environment variable that contains the list of namespaces allowed to host cluster config
 	// instances
 	ArgoCDClusterConfigNamespacesEnvVar = "ARGOCD_CLUSTER_CONFIG_NAMESPACES"
+
+	// ArgoCDLabelSelectorEnvVar is an environment variable that contains the labels used for selective instance reconilliation.
+	ArgoCDLabelSelectorEnvVar = "ARGOCD_LABEL_SELECTOR"
```

Comment:
Just to confirm, when common/TOBEREMOVED.go gets removed, `ArgoCDLabelSelectorKey` will be replaced by `ArgoCDLabelSelectorEnvVar` right? There would be another rebase PR to update all the constants to use the new name?

Reply:
yes, we'll handle those cases later